cypress/integration/my-orcid/my-orcid-affiliations.e2e.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+describe.only('Affiliations' + runInfo(), () => {
+  describe.only('Employment' + runInfo(), () => {
+    it('show an option to add more items', () => {})
+    it('show an option to edit items', () => {})
+    it('display a user with no items', () => {
+      // TODO ??
+    })
+
+    describe('display a user with an item added by the same user', () => {
+      it('show the number of works as 1', () => {})
+      it('display the item privacy', () => {})
+      it('show the item details', () => {})
+      it('opens external URLs on a separate tab', () => {})
+      it('show the item edit button', () => {})
+      it('can be deleted', () => {})
+    })
+
+    describe('display a user with an item added with the API', () => {
+      it('show the number of items on the title as 1', () => {})
+      it('show the item source', () => {})
+      it('show the item "make a copy and edit" button', () => {})
+      it('can be deleted', () => {})
+    })
+
+    describe('display a user with an item added with multiple sources', () => {
+      it('show the number of items on the title as 1', () => {})
+      it('show the item multiple sources', () => {})
+      it('expand and show the multiple sources of the same item', () => {
+        // TODO ??
+      })
+    })
+
+    describe('display a user with many items', () => {
+      // TODO ??
+      it('show the number of items on the title as N', () => {})
+      it('can sort the items by the end date', () => {})
+      it('can sort the items by start date', () => {})
+      it('can sort the items by start title', () => {})
+    })
```

Comment:
@dandineen The sort menu that best matches the one on production affiliations is shown for works displays

![image](https://user-images.githubusercontent.com/2119626/109048812-1fc6d700-769d-11eb-894b-b48860225251.png)

But it only shows order by `start date`, `end date and `alphabetically`, and not the `start date reverse`, `end date reverse` and  `alphabetically`.  Is the reverse something we want to remove?

Reply:
I am planning to add a sort arrow like the one use [Here](https://material.angular.io/components/sort/overview) on the Table with sorting example 

![image](https://user-images.githubusercontent.com/2119626/109049802-4b968c80-769e-11eb-970f-ac0e0ab6bd95.png)

![image](https://user-images.githubusercontent.com/2119626/109049972-81d40c00-769e-11eb-86c8-2eaa04373adb.png)

Reply:
We don't currently give the `reverse` options anywhere do we?

Oh, hold on! If you click the same sort filter a second time it reverses the sorting direction. The icons are so small I didn't realise they were changing!

I think the _behaviour_ should stay the same but use those larger, clearer and more obvious arrow icons as you suggested above.